client/create_deployment.go

```diff
@@ -145,7 +152,7 @@ func CreateDeployment(cli *Cli, commandOptions *CommandOptionsCreateDeployment)
 	}
 
 	//Call rest client
-	deploymentResponse := DeplyomentResponse{}
+	deploymentResponse := DeploymentResponse{}
 	if err := cli.Client.DoRequest(http.MethodPost, url, &deploymentRequest, &deploymentResponse); err != nil {
 		log.Fatal("Error rest call: ", err)
```

Comment:
Better return error instead of os.exit(1). The client should not exit, the cli can.